### ISSUE TITLE
[Accessibility]  Bug: Missing alternative text

### ISSUE THREAD

**Issue by ajbrookshire** [ajbrookshire/QA-Test#2]
## Environment
Device: DESKTOP
OS: Windows 10 Home
App version: n/a

### Expected Behavior
What is expected behavior?
I expected alternative text for the selected image. 

What is actual behavior?
No alternative text is available for the selected image.

**Steps to Reproduce**
1. Navigate to url www.nutrish.com
2. Select WAVE Extension
3. View text shown below

Screenshots/screen recordings
![accessibility_1](https://user-images.githubusercontent.com/99623750/165130557-af019936-b74b-4048-adda-919629dd019d.png)
